src/components/HamburgerMenu/SidebarNav.js

```diff
@@ -0,0 +1,49 @@
+import React, { useRef } from 'react';
```

Comment:
Opening the menu looks really nice, but when I click it again to close it, it takes around 1 second till it is closed. I think it is related to very big circle. Try to fix it